app/main/routes_zseries_api.py

```diff
@@ -373,7 +380,8 @@ def update_session_log(token, body):
     if active_session not in events:
         events[active_session] = []
     
-    events[active_session].append(body['StepName'])
+    if active_brew_sessions[uid].recovery != body['StepName']:
+        events[active_session].append(body['StepName'])
```

Comment:
@chiefwigms this actually fixes the event publishing by only adding `event` to the data when the `StepName` pushed changes whereas in Zymatic or P/C/Pro it comes as a unique event from the device

Reply:
![image](https://user-images.githubusercontent.com/2158627/84717150-059bfd00-af43-11ea-98e0-eef0d7da8407.png)